tracing/tracer.go

```diff
@@ -49,53 +50,19 @@ func CreateTracer(
 	cfg *config.Configuration,
 	serviceName string,
 	logger *zap.Logger,
-	statsReporter tally.CachedStatsReporter,
+	scope tally.Scope,
 ) (opentracing.Tracer, io.Closer, error) {
-	var reporter *jaegerReporter
 	if cfg == nil || !cfg.Disabled {
-		cfg = loadAppConfig(cfg, logger)
-		// TODO: Change to use the right stats reporter
-		reporter = &jaegerReporter{
-			reporter: statsReporter,
-		}
+		cfg = loadAppConfig(cfg)
 	}
-	return cfg.New(serviceName, reporter)
+	jMetrics := jtally.Wrap(scope)
```

Comment:
tiniest of nits: this seems like a really weird variable name to me. id either go really short (`jm`) or explain a bit more (`jaegerMetrics := ...`)